packages/front-end/src/app/components/EGHeader.vue

```diff
@@ -57,7 +57,7 @@
           </ULink>
           <ULink
             v-if="useUserStore().canManageOrgs"
-            :to="useUserStore().isSuperuser ? '/admin/orgs' : '/orgs'"
+            :to="'/orgs'"
```

Comment:
String can be simply passed as `to="/orgs"`

Reply:
done 👍